framework/db/ActiveRecord.php

```diff
@@ -291,12 +303,18 @@ public static function tableName()
      */
     public static function getTableSchema()
     {
-        $schema = static::getDb()->getSchema()->getTableSchema(static::tableName());
-        if ($schema !== null) {
-            return $schema;
-        } else {
-            throw new InvalidConfigException("The table does not exist: " . static::tableName());
+        static $tableSchema = null;
+
+        if ($tableSchema === null) {
+            $schema = static::getDb()->getSchema()->getTableSchema(static::tableName());
```

Comment:
This eliminates the ability of refreshing table schema in case some changes to table structure have been made.